PlayCover/Views/Settings/IPASourceSettings.swift

```diff
@@ -102,11 +103,16 @@ struct IPASourceSettings: View {
 
 struct SourceView: View {
     var source: SourceData
+    @State var isEnabled: Bool
     @State var showingPopover = false
 
     var body: some View {
         HStack {
-            Text(source.source)
+            Toggle("", isOn: $isEnabled).onChange(of: isEnabled) { value in
```

Comment:
`.onChange` should be on a newline after toggle, and preferably at the bottom of the view, as that is most common throughout the codebase. Also, the source text should be with the toggle instead of in its own `Text`.